{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -72,27 +72,26 @@ testpaths = [
 [tool.ruff]
 fix = true
 force-exclude = true
-ignore = [
+lint.ignore = [
     "COM812", # trailing commas (ruff-format recommended)
     "D203", # no-blank-line-before-class
     "D212", # multi-line-summary-first-line
     "D407", # removed dashes lines under sections
     "D417", # argument description in docstring (unreliable)
     "ISC001", # simplify implicit str concatenation (ruff-format recommended)
 ]
-per-file-ignores = {"tests*" = [
+lint.per-file-ignores = {"tests*" = [
     "INP001",
     "S101",
 ]}
-select = [
+lint.select = [
     "ALL",
 ]
-target-version = "py{{cookiecutter.min_python_version | replace('.', '')}}"
```

Comment:
[`requires-python` is recommended over `target-version`](https://docs.astral.sh/ruff/settings/#target-version), and we don't need both.